src/module.c

```diff
@@ -1032,11 +1032,29 @@ int isCommandNameValid(const char *name) {
     return 1;
 }
 
+/* Helper for commandFlagsFromString(). Turns a string representing command
+ * flags into the ACL Category Flags.
+ *
+ * Returns 'REDISMODULE_OK' if acl category flag is recognized or
+ * returns 'REDISMODULE_ERR' if not recognized  */
+int matchAclCategoriesFlags(char *flag, int64_t *acl_categories){
+    if (flag[0] == '@'){
+        uint64_t this_flag = ACLGetCommandCategoryFlagByName(flag + 1);
+        if (this_flag) {
+            *acl_categories |= (int64_t) this_flag;
+            serverLog(LL_NOTICE,"Recognized acl category flag %s on module load", flag);
```

Comment:
why do we need / want that log?
maybe the log level should be reduced, or if we really want to have some warning present when modules use this feature, maybe a more official introspection feature is needed (in INFO or MODULE command)